chat-plugins/info.js

```diff
@@ -532,7 +532,7 @@ var commands = exports.commands = {
 
 		for (var cat = 0; cat < categories.length; cat++) {
 			var search = categories[cat];
-			if (!searches[search]) continue;
+			if (!(searches in search)) continue;
```

Comment:
My bad, the initial implementation was correct, but that was while I was working on `ds` on my own server, and I copied it incorrectly.